docs/src/components/hardware-list.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+
+import {
+  Board,
+  HardwareMetadata,
+  Interconnect,
+  Shield,
+} from "../hardware-metadata";
+
+interface HardwareListProps {
+  items: HardwareMetadata[];
+}
+
+type ElementOrString = JSX.Element | string;
+
+function itemHasMultiple(item: HardwareMetadata) {
+  return (
+    (item.type == "board" || item.type == "shield") &&
+    (item.siblings?.length || 1) > 1
```

Comment:
```suggestion
    item.siblings?.length > 1
```
The above code works, but `undefined > 1` is also false, so you could simplify this